docs/source/ecosystem/flash.rst

```diff
@@ -0,0 +1,76 @@
+Lightning Flash
+===============
+
+`Lightning Flash <https://lightning-flash.readthedocs.io/en/latest/>`_ is a high-level deep learning framework for fast prototyping, baselining, fine-tuning, and solving deep learning problems.
```

Comment:
```suggestion
`Lightning Flash <https://lightning-flash.readthedocs.io/en/stable/>`_ is a high-level deep learning framework for fast prototyping, baselining, fine-tuning, and solving deep learning problems.
```
to be aligned with later installing package... otherwise you would need to install from source